docs/getting-started/overview.md

```diff
@@ -1,17 +1,238 @@
-- [Add time-series data][2]
-- [Query Data with SQL][3]
-- [Process Data with Python][4]
-- [Visualize data][5]
+## Install GreptimeDB
+You can try out GreptimeDB with our test builds released on [Download page](https://greptime.com/download). 
 
-We create a time-series table to store system metrics from hosts in IDC as an example.
+We use the simplest configuration for getting started. For a comprehensive list of configuration options available in GreptimeDB, see the [configuration documentation](/user-guide/operations/configuration.md).
+
+### Linux & Mac
+
+For Linux and macOS users, you can download the latest build of the `greptime` binary by using the following commands:
+
+```shell
+curl -L https://raw.githubusercontent.com/GreptimeTeam/greptimedb/develop/scripts/install.sh | sh
+```
+
+When the download is complete, the binary file `greptime` will be stored in your current directory. 
+
+You can run the GreptimeDB in standalone mode:
+
+```shell
+./greptime standalone start
+```
+
+### Docker
+Make sure the [Docker](https://www.docker.com/) is already installed. If not, you can follow the official [documents](https://www.docker.com/getting-started/) to install Docker.
+
+```shell
+docker run -p 4000-4004:4000-4004 \
+-p 4242:4242 -v "$(pwd)/greptimedb:/tmp/greptimedb" \
+--name greptime --rm \
+greptime/greptimedb standalone start \
+--http-addr 0.0.0.0:4000 \
+--rpc-addr 0.0.0.0:4001 \
+--mysql-addr 0.0.0.0:4002 \
+--postgres-addr 0.0.0.0:4003 \
+--prom-addr 0.0.0.0:4004 \
+--opentsdb-addr 0.0.0.0:4242
+```
+
+The data will be stored in the `greptimedb/` directory in your current directory.
+
+If you want to use another version of GreptimeDB's image, you can download it from our [GreptimeDB Dockerhub](https://hub.docker.com/r/greptime/greptimedb).
+
+**Note**:
+
+If you are using a Docker version lower than [v23.0](https://docs.docker.com/engine/release-notes/23.0/), you may experience problems with insufficient permissions when trying to run the command above, due to a [bug](https://github.com/moby/moby/pull/42681) in the older version of Docker Engine.
+
+You can:
+
+1. Set  `--security-opt seccomp=unconfined`, for example:
+
+   ```shell
+   docker run --security-opt seccomp=unconfined -p 4000-4004:4000-4004 \
+   -p 4242:4242 -v "$(pwd)/greptimedb:/tmp/greptimedb" \
+   --name greptime --rm \
+   greptime/greptimedb standalone start \
+   --http-addr 0.0.0.0:4000 \
+   --rpc-addr 0.0.0.0:4001 \
+   --mysql-addr 0.0.0.0:4002 \
+   --postgres-addr 0.0.0.0:4003 \
+   --prom-addr 0.0.0.0:4004 \
+   --opentsdb-addr 0.0.0.0:4242
+   ```
+
+2. Upgrade the Docker version to v23.0.0 or higher;
+
+
+## Connect
+
+GreptimeDB supports [multiple protocols](/user-guide/clients.md). We use MySQL client here for simplicity.
+
+```sql
+mysql -h 127.0.0.1 -P 4002
+```
+
+## Create table
+
+Let's start by creating the `system_metrics` table which contains system resource metrics, including CPU/memory/disk usage. The data is scraped every 5 seconds.
```

Comment:
We can add a note here saying we support automatically creating tables when using InfluxDB, Prometheus remote write, and OpentsDB protocol.

Reply:
we can add a link to Table Management where described creating table automatically.

Reply:
@nicecui All right, that would be cool. I am just concerned that the users may think we are a strong schema database, but in fact, is not. So maybe we can say that we are a schema-less database and link to table management.

Reply:
A note with the table management link has added after creating table in getting started chapter, and the automatically creating has been emphasized in the table management chapter.